src/utils/displayFunctions.ts

```diff
@@ -122,7 +122,7 @@ export const makeDisplayFunctions = (brandToInfo: Map<Brand, BrandInfo>) => {
     return new Intl.DateTimeFormat(navigator.language, {
       timeStyle: 'medium',
       dateStyle: 'short',
-    }).format(new Date(Number(price.timestamp) * 1000));
+    }).format(new Date(Number(price.timestamp.absValue) * 1000));
```

Comment:
This could be backward compatible by using the `absValue` helper in agoric/time. (Named something like that)

If there are other time values this may be a good time(!) to wrap them with the accessor function (because it supports both formats)